packages/story-editor/src/components/richText/karma/inlineStyleOverride.karma.js

```diff
@@ -320,7 +318,7 @@ describe('Inline style override', () => {
     const actual = getTextContent();
     const expected = '<span style="font-weight: 700">Full in some text</span>';
     expect(actual).toBe(expected);
-    await data.fixture.snapshot('"Full in some text" in bold');
+    await data.fixture.snapshot('"Fill in some text" in bold');
```

Comment:
But it clearly expects the text to read `"Full"` though (note [line 312](https://github.com/google/web-stories-wp/pull/10097/files#diff-0d6a45c026c2f74906a8515916c3aa73c24f51c32ed593b47acb93f14bbb55d3R312) changing the "i" to a "u").

(it should perhaps be a more significant text change to clearly show that this is not a typo)

Reply:
Oops, I'll revert that, thanks!